### ISSUE TITLE
Update required

### ISSUE THREAD

**Issue by darkolacen** [Errorname/prisma-multi-tenant#17]
Prisma 2 updated its commands. So this package is currently not usable.
Error it throws:

```
Unknown Error!
Error: Command failed: prisma2 generate
Error: The generator provider photonjs with the corresponding package @prisma/photon has been deprecated.
The provider has been renamed to prisma-client-js and the package to @prisma/client".
@prisma/client now exposes "PrismaClient instead of Photon. Please update your code accordingly
```

**Comment by Errorname** [comment]
Hi! Thank you for using prisma-multi-tenant 🙏 
This error is thrown because the last Prisma Preview (preview20 released yesterday) changed the name of the generator. Unfortunately, I am unavailable until sunday. But I'll try to update as soon as possible 😊 (Probably around mid-week next week)

**Comment by Errorname** [comment]
Hi there! 
I just published `prisma-multi-tenant@2.0.0-alpha17` that is compatible with the new `prisma2@2.0.0-preview020.3` version 🙂
Dont hesitate to ping me here if there is still issues on this version!
